bin/public/js/term.js

```diff
@@ -1328,7 +1328,8 @@ Terminal.prototype.refresh = function(start, end) {
                 out += 'font-weight:bold;';
               }
               // See: XTerm*boldColors
-              if (fg < 8) fg += 8;
+              if(this.options.boldAsBright > 0 && fg < 8)
```

Comment:
> If we really want to stay in sync with the original projects better

That's a big problem in open-source. Lots of folks create nice projects, but when you contribute to them, they just don't merge your contributions (e.g. they are just busy, miss the PRs or just forget, like I just did).

For example I opened https://github.com/chjj/term.js/pull/89 in September and it got merged few weeks ago. But since September I was using the patched version because that bug was fixed.

When using `npm` we anyway can change the dependency to `owner/fork` when such things happen. :grin: